contracts/contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -1300,7 +1300,7 @@ namespace eosiosystem {
                                             double additional_shares_delta = 0.0, double shares_rate_delta = 0.0 );
          bool does_satisfy_stake_requirement( const name& producer ) const;
          bool is_block_producer( const name& producer ) const;
-         
+
```

Comment:
Maybe revert this changes? File included in PR only because of spaces seems weird